packages/@icon-magic/distribute/src/distribute-svg.ts

```diff
@@ -33,8 +35,20 @@ export async function distributeSvg(
   const spriteNames: SpriteConfig = {};
   const promises: void[] = [];
   for (const icon of icons) {
-    LOGGER.debug(`calling distributeSvg on ${icon.iconName}: ${icon.iconPath}`);
+    LOGGER.debug(`calling distributeSvg on ${icon.iconName}: ${icon.iconPath} with colorScheme: ${colorScheme}`);
     const assets = getIconFlavorsByType(icon, 'svg');
+
+    // Further filter the icons by matching the assets's colorScheme to the commander option --colorScheme
+    const assetsByColorScheme = assets.filter(asset => {
+      if (asset.colorScheme) {
+        return colorScheme.includes(asset.colorScheme);
+      } else if (asset.colorScheme === undefined){
```

Comment:
```suggestion
      } else if {
```

allow for `undefined` and other null values.

Reply:
I think just the else should be enough. we already know it's not a truthy value from the first if statement.

Reply:
Oh nice, my only question now is if we need the return false?  I believe not right? If that is correct then we don't even need the else?
```
    const assetsByColorScheme = assets.filter(asset => {
      if (asset.colorScheme) {
        return colorScheme.includes(asset.colorScheme);
      } 
      // Light variants can either have colorScheme: `light`, null, or undefined
      return colorScheme.includes('light');
    });
    ```

Reply:
Tested and this works fine so I'm changing it to this^. Thx.